### PR TITLE
Replace hardcoded jenkins string

### PR DESCRIPTION
**Pull request by v1v**
Some leftovers which, IMO, should use the variable `@@ARTIFACTNAME@@`

@reviewbybees, @svanoort @jtnord

### CODE REVIEW

File: rpm/build/SOURCES/jenkins.sysconfig.in [hunk 1]
```diff
@@ -2,7 +2,7 @@
 ## Description: @@SUMMARY@@
 ## Type:        string
 ## Default:     "~~HOME~~"
-## ServiceRestart: jenkins
+## ServiceRestart: @@ARTIFACTNAME@@
```
Comment (jtnord) [inline rpm/build/SOURCES/jenkins.sysconfig.in]: If you do this then service start Jenkins does not start jenkins but `@@ARTIFACTNAME@@` which if it is not jenkins becomes unexpected.

**Comment by daniel-beck** [comment]
Wasn't there an objection previously related to service file name inconsistency? What happened to that?

**Comment by oleg-nenashev** [comment]
@daniel-beck @v1v 
w.r.t  https://github.com/jenkinsci/packaging/pull/108#issuecomment-392295778 and ....

> Some leftovers which, IMO, should use the variable @@ARTIFACTNAME@@

Could you please add links to the related discussion?

**Comment by v1v** [comment]
That particular section was removed with https://github.com/jenkinsci/packaging/commit/581cc9f8f3d86fbc744a796d3129b5f40ebdb228#diff-db856906108ddd6a892a09c67f334b36

And it seems other distributions such as `suse` does use the `@@ARTIFACTNAME@@`:
- https://github.com/jenkinsci/packaging/blob/c4c7373d8a512073618501cdf1b280041fa34086/suse/build/SOURCES/jenkins.sysconfig.in#L5

It could potentially leave it as is although whoever would like to customize the name of the service, in case, they wanna use this packaging system, they should hardcode it. 

No strong opinions though

**Comment by v1v** [comment]
what's the status of this PR? Are there any other requirements to be addressed?

**Comment by oleg-nenashev** [comment]
I think we are fine here

**Pull request merged**